Facepunch.Steamworks/SteamNative/SteamNative.SteamUGC.cs

```diff
@@ -284,9 +284,12 @@ public bool GetQueryUGCStatistic( UGCQueryHandle_t handle /*UGCQueryHandle_t*/,
 		}
 		
 		// uint
-		public uint GetSubscribedItems( PublishedFileId_t* pvecPublishedFileID /*PublishedFileId_t **/, uint cMaxEntries /*uint32*/ )
+		public uint GetSubscribedItems( PublishedFileId_t[] pvecPublishedFileID /*PublishedFileId_t **/, uint cMaxEntries /*uint32*/ )
```

Comment:
This file is all generated code, we shouldn't be editing it manually - but install fixing any problems with the generator (if any)